### DISCUSSION TITLE
Difficulty using curobo.wrap.reacher.motion_gen with custom Xarm configuration

### DISCUSSION THREAD

**Discussion by D-1shu** [NVlabs/curobo#188] [Software Q&A]
I'm encountering an issue while using the curobo.wrap.reacher.motion_gen function with a custom Xarm configuration file. I can successfully solve for the final joint positions using curobo.wrap.reacher.ik_solver, but attempts to generate trajectories using motion_gen fail, are there specific steps I need to take when using a custom configuration file with motion_gen?
[xarm_r.zip](https://github.com/NVlabs/curobo/files/14592037/xarm_r.zip)

**Comment by nbubis** [comment]
Hi, did you end up solving this? would you be able to post what worked for you?
